### PR TITLE
docs: modify the descendant spelling and remove multiple times parent in DI guide

### PR DESCRIPTION
**Pull request by nchetankumar**
Modified the descendant spelling  and removed multiple times parent's word

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by brandonroberts** [comment]
Thanks for the PR @nchetankumar! You must [Sign the CLA](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#cla) before this PR can move forward.

**Comment by nchetankumar** [comment]
Hey Brandon,

It looks like i have already signed the CLA.Please ref the below screenshot,
[image: image.png]

File: aio/content/guide/dependency-injection.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ Child modules and component injectors are independent of each other, and create
 Thanks to [injector inheritance](guide/hierarchical-dependency-injection),
 you can still inject application-wide services into these components.
 A component's injector is a child of its parent component's injector,
-and a descendent of its parent's parent's injector, and so on all the way back to the application's _root_ injector. Angular can inject a service provided by any injector in that lineage.
+and a descendant of its parent's injector, and so on all the way back to the application's _root_ injector. Angular can inject a service provided by any injector in that lineage.
```
Comment (jbogarthyde) [inline aio/content/guide/dependency-injection.md]: How about: 
A component's injector is a child of its parent component's injector, and inherits from all ancestor injectors all the way back to the application's _root_ injector.

**Review by jbogarthyde — CHANGES_REQUESTED** [review]
I think it's clearer this way.

**Review by brandonroberts — APPROVED** [review]

**Review by jbogarthyde — APPROVED** [review]

**Pull request closed without merging**